logger_test.go

```diff
@@ -0,0 +1,36 @@
+package analytics
+
+import (
+	"bytes"
+	"errors"
+	"log"
+	"testing"
+)
+
+// This test ensures that the interface doesn't get changed and stays compatible
```

Comment:
Oh I see what you mean, just moving it to the global scope instead of having a test that actually get run?

Yeah that would work.